apps/etterlatte-utbetaling/src/main/kotlin/no/nav/etterlatte/utbetaling/iverksetting/oppdrag/OppdragMapper.kt

```diff
@@ -98,6 +99,13 @@ object OppdragMapper {
                             sats = it.beloep
                             fradragTillegg = OppdragslinjeDefaults.FRADRAG_ELLER_TILLEGG
                             typeSats = OppdragslinjeDefaults.UTBETALINGSFREKVENS
+
+                            // For å teste ifm etteroppgjøret
+                            // TODO: bytte ut sakid med riktig
+                            if (it.sakId.value == SakId(123).sakId && it.periode.fra.year == 2024) {
```

Comment:
Burde man ha med miljø her også for å unngå en potensiell smell i prod?

Reply:
Ja, det er nok lurt